pyhuman/app/workflows/browse_web.py

```diff
@@ -49,3 +72,29 @@ def _load_website_list():
             for line in f:
                 wordlist.append(line)
         return wordlist
+
+    def _navigate_website(self):
+        # Browse the currently loaded website with a random amount of clicks
+        navigation_clicks = random.randrange(0, self.max_navigation_clicks)
+        for num_click in range(1, navigation_clicks):
+            clickables = self.driver.driver.find_elements(By.TAG_NAME, ("a"))
+            # If there's nothing to click, stop navigating this page
+            if len(clickables) == 0:
+                print(f"... {num_click}. No clickable elements were found")
+                return
+            # Navigate to a random url on the page
+            else:
+                clickable = random.choice(clickables)
+                url = clickable.get_attribute("href")
+                if url is None:
+                    print(f"... {num_click}. Invalid URL ")
+                    continue
+                try:
+                    self.driver.driver.get(url)
```

Comment:
One bit of feedback after testing, it may be a good idea to have it sleep for a bit in between each navigation. It seemed to navigate between webpages very, very quickly. It may be more humanlike if it holds on a page for a variable period of time before clicking off of it.

Reply:
Good point!